.github/workflows/ci.yaml

```diff
@@ -1013,3 +1013,24 @@ jobs:
     uses: ./.github/workflows/build-msi.yml
     if  : github.event_name == 'pull_request' && contains(github.event.pull_request.body, '[test_msi]')
     # TODO: ADD A JOB THAT DEPENDS ON THIS TO TEST THE MSI
+    
+  build-sdk-package:
+    uses: ./.github/workflows/build-sdk.yml
+    with:
+      java-version: 8
+
+  build-chocolatey-package:
+    uses: ./.github/workflows/build-chocolatey.yml
+    needs: [ build-sdk-package ]
+    with:
+      version: 3.6.0-local # TODO: FIX THIS
```

Comment:
Shouldn't this be fixed before the merge? There is another usage of that in the next job. 
I guess this value would need to be frequently updated, so it should be documented somewhere (maybe a mention in the project/Build.scala baseVersion that changing one value should also update ci file)

Reply:
Maybe `build-chocolatey.yml` can expose version as output. That way we would never forget about synchronizing the versions in both of the jobs

Reply:
actually, this value when testing doesn't matter. I just have to give a version number to the package we are currently building (this package is local to the build). Nevertheless, it would be great if we have something that is correct and computes it directly without doing anything else.

Reply:
I've added a todo to check later what to change this value to but it is not at all a blocker